pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
Essentially being an ETL tool, going with that avoids all these complexities we are having with the language. I should have thought about it before!